src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -100,3 +104,46 @@ def validate_query_fields(namespace):
                 if field.lower() == query_field.name.lower():
                     fields.append(query_field)
         namespace.fields = fields
+
+
+def validate_feature_query_fields(namespace):
+    if namespace.fields:
+        fields = []
+        for field in namespace.fields:
+            for feature_query_field in FeatureQueryFields:
+                if field.lower() == feature_query_field.name.lower():
+                    fields.append(feature_query_field)
+        namespace.fields = fields
+
+
+def validate_filter_parameters(namespace):
+    """ Extracts multiple space-separated filter paramters in name[=value] format """
+    if isinstance(namespace.filter_parameters, list):
```

Comment:
If namespace.filter_parameters is not list, you will not validate it, right? Will it cause any error for users? If yes, can we throw valid error message?

Reply:
We are mimicking the CLI inbuilt tags_type for this validation. Since filter_parameters_arg_type has defined nargs=*, namespace.filter_parameters will always be a list. 


---
In reply to: [335412938](https://github.com/Azure/azure-cli/pull/10774#discussion_r335412938) [](ancestors = 335412938)